### PR TITLE
Add README.md + general polishes

### PR DESCRIPTION
**Pull request by feliperodriguess**
## Describe Changes

Changes made to the package: **sdk**.

I made it better by doing ...

- Add SDK README.md file with some guides
- Some general polishes on unnecessary properties/methods

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [x] Code styles have been enforced
- [x] All tests are passing

### CODE REVIEW

**Review by sogoiii — CHANGES_REQUESTED** [review]
Can we update to the latest 9 version of pnpm. You will have to update the `package.json` and the `.tool-versions` file.

File: README.md [hunk 1]
```diff
@@ -1,3 +1,141 @@
...
+
+To use Holograph SDK, all you need to do is install the
+`@holographxyz/sdk` package and its peer dependencies:
+
+```sh
+# with pnpm
+$ pnpm add @holographxyz/sdk @holographxyz/networks @holographxyz/environment
+
+# with npm
+$ npm i @holographxyz/sdk @holographxyz/networks @holographxyz/environment
```
[the remark this answers is not in the document]
Comment (feliperodriguess) [reply README.md:52]: it works, we use yarn on our FE

File: packages/sdk/package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "@holographxyz/sdk",
-  "version": "0.1.0-alpha.80",
+  "version": "0.1.0-alpha.81",
```
[the remark this answers is not in the document]
Comment (feliperodriguess) [reply packages/sdk/package.json:3]: sure

**Comment by feliperodriguess** [comment]
> Can we update to the latest 9 version of pnpm. You will have to update the `package.json` and the `.tool-versions` file.

Done

**Review by sogoiii — APPROVED** [review]

**Pull request merged**